plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -174,7 +174,7 @@
       image_url:
         required: false
         description:
-          - The URL od the image the insert or eject
+          - The URL of the image to insert or eject
```

Comment:
While we're at it:
```suggestion
          - The URL of the image to insert or eject.
```
:)

Reply:
I did not make that change for consistency reasons. All the descriptions in this file do not end in a full stop, so either they should be all changed or none - in my opinion.

Reply:
Ok. I guess someone will have to add them all then (since descriptions should always contain full sentences).

Reply:
If they should all have full stops at the end I can make another PR for that and do some other language improvements (like drop "The" from descriptions like "The password for [...]")

Reply:
That would be great!